main.js

```diff
@@ -0,0 +1,3 @@
+import { default as getRandomNumber } from "./module1/index.js";

```

Comment:
Because of the default export in index it throws `file:///C:/Users/attila_gal/DEV/nodejs-mentoring/main.js:1
import { getRandomNumber } from "./module1/index.js";
         ^^^^^^^^^^^^^^^
SyntaxError: The requested module './module1/index.js' does not provide an export named 'getRandomNumber'
    at ModuleJob._instantiate (node:internal/modules/esm/module_job:134:21)
    at async ModuleJob.run (node:internal/modules/esm/module_job:217:5)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async asyncRunEntryPointWithESMLoader (node:internal/modules/run_main:123:5)`
the following` import getRandomNumber  from "./module1/index.js";` would work but than I am importing the whole module. Its an interesting thing to read about ty. :)

Reply:
That's weird, we might check this later.